### ISSUE TITLE
Hint on training

### ISSUE THREAD

**Issue by hcl14** [ShineChen1024/oms-Diffusion#31]
Hi, thanks very much for the great project! 

I am working on my own training script, and I based it on controlnet training - I do noise prediction for random time step and compare it with real noise. Trainable `ref_unet` is inferenced for time step 0 always and passes its attention maps through dictionary to the main unet. Weights of the main unet are frozen. I do training in fp16 precision with 8-bit AdamW optimizer.

https://github.com/hcl14/oms-Diffusion/blob/cdbbbe9060027b51fef0ac35f5b1d06342c6e496/train_attempt_my_controlnet_agnostic_mask.py#L171

I also added controlnet-openpose for guidance and cloth-agnostic masks applied to latent in attempt to make work easier. I also use gradient accumulation to imitate big batch size. 

Currently the training diverges very quickly with lr=1e-5 and more slowly with lr=1e-6 and starts to produce noisy image. 

I know you plan to release code later, but perhaps you can give me some hints on it?

Thank you very much!
